octopoes/tools/xtdb_client.py

```diff
@@ -119,7 +119,8 @@ def tx_log(
         return res.json()
 
     def submit_tx(self, transactions: list[str]) -> JsonValue:
-        res = self._client.post("/submit-tx", json={"tx-ops": transactions})
+        data = f"{{\"tx-ops\": [{", ".join(transactions)}]}}"
```

Comment:
Other than that vulture doesn't like this, I think this is also more readable:

```suggestion
        joined_transactions = ", ".join(transactions)
        data = f"{{\"tx-ops\": [{joined_transactions}]}}"
```

Reply:
Is it more readable because vulture fails parsing correct code?

Reply:
Also interestingly, the vulture I run locally through precommit is perfectly happy with this code.

Reply:
maybe a version difference?
I'm perfectly happy with the inital version.